lib/google_maps_service/apis/roads.rb

```diff
@@ -104,6 +104,42 @@ def snapped_speed_limits(path)
                  custom_response_decoder: method(:extract_roads_body))
     end
 
+    # Returns the Nearest road segments for provided points in a path.
+    #
+    # Takes up to 100 independent coordinates, and returns the closest road segment for each point.
+    # The points passed do not need to be part of a continuous path.
+    #
+    # @example Single point snap
+    #   results = client.nearest_roads([40.714728, -73.998672])
+    #
+    # @example Multi points snap
+    #   path = [
+    #       [-33.8671, 151.20714],
+    #       [-33.86708, 151.20683000000002],
+    #       [-33.867070000000005, 151.20674000000002],
+    #       [-33.86703, 151.20625]
+    #   ]
+    #   results = client.nearest_roads(path)
+    #
+    # @param [Array] path The path to be match to nearest road segment. Array of latitude/longitude pairs.
```

Comment:
The param name should be `points` as in the method declaration. 

I think "match" sounds not really fit here, any other words?
Or, you can also move:

```
    # Takes up to 100 independent coordinates, and returns the closest road segment for each point.
    # The points passed do not need to be part of a continuous path.
```

to `points` description.